docs/guides/0-web-verification.md

```diff
@@ -0,0 +1,217 @@
+
+The verifier server handles the complex cryptographic verification, while your Android app focuses on secure credential presentation.
+
+## **Implementation Steps**
+
+### **1. Add CredmanActivity**
+
+Create a new activity extending `CredentialManagerPresentmentActivity `provided by Multipaz. This activity is launched when a browser requests credentials via the W3C DC API.
+
+```
+package org.multipaz.getstarted
+
+import androidx.compose.runtime.Composable
+import multipazgettingstartedsample.composeapp.generated.resources.Res
+import org.multipaz.compose.digitalcredentials.CredentialManagerPresentmentActivity
+
+class CredmanActivity : CredentialManagerPresentmentActivity() {
+    @Composable
+    override fun ApplicationTheme(content: @Composable (() -> Unit)) {
+        App()
+        content()
+    }
+
+    override suspend fun getSettings(): Settings {
+        val app = App.getInstance()
+        app.init()
+        return Settings(
+            appName = app.appName,
+            appIcon = app.appIcon,
+            promptModel = App.promptModel,
+            documentTypeRepository = app.documentTypeRepository,
+            presentmentSource = app.presentmentSource,
+            privilegedAllowList = Res.readBytes("files/privilegedUserAgents.json").decodeToString()
+        )
+    }
+}
+```
+
+Refer to **[**the CredmanActivity file**](https://github.com/openwallet-foundation/multipaz-samples/blob/af7f2adea05fd2ca531cdc50824e556b88b770fd/MultipazGettingStartedSample/composeApp/src/androidMain/kotlin/org/multipaz/getstarted/CredmanActivity.kt)** for the complete example.
+
+### **2. Update AndroidManifest.xml**
+
+Register `CredmanActivity` in your manifest and declare intent filters for the Credential Manager API actions.
+
+```
+<activity
+    android:name=".CredmanActivity"
+    android:configChanges="orientation|screenSize|screenLayout|keyboardHidden|mnc|colorMode|density|fontScale|fontWeightAdjustment|keyboard|layoutDirection|locale|mcc|navigation|smallestScreenSize|touchscreen|uiMode"
+    android:exported="true"
+    android:launchMode="singleInstance"
+    android:theme="@android:style/Theme.Translucent">
+    <intent-filter>
+        <action android:name="androidx.credentials.registry.provider.action.GET_CREDENTIAL" />
+        <action android:name="androidx.identitycredentials.action.GET_CREDENTIALS" />
+        <category android:name="android.intent.category.DEFAULT" />
+    </intent-filter>
+</activity>
+```
+
+#### **What does this do?**
+
+* This registers your app as a credential provider for browser and web app requests using the W3C DC API.
+
+Refer to the [**sample Manifest code**](https://github.com/openwallet-foundation/multipaz-samples/blob/af7f2adea05fd2ca531cdc50824e556b88b770fd/MultipazGettingStartedSample/composeApp/src/androidMain/AndroidManifest.xml#L55-L67) for context.
+
+### **3. Add Privileged User Agents JSON**
+
+Create a JSON file listing all trusted browser apps and their signature fingerprints.
```

Comment:
Please expand a bit the need for this step. For instance:

`This JSON defines the trusted browsers (by package name and certificate fingerprint) that are allowed to handle credential requests, ensuring only verified apps can participate.`

Reply:
I have included a "what does this do" section below this - can you please check that and LMK if I should further edit this?